sinch/domains/verification/enums.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class VerificationMethod(Enum):
+    SMS = "sms"
+    FLASHCALL = "flashCall"
```

Comment:
Docs and OAS file specify this method as `flashCall`

Reply:
In the end I'm transforming this property into more Python friendly snake_case anyway:

@dataclass
class StartFlashCallInitiateVerificationResponse(StartVerificationResponse):
    flash_call: dict

Reply:
The enum value (https://github.com/sinch/sinch-sdk-python/blob/DEVEXP-205-186-verification-API/sinch/domains/verification/enums.py#L6) is still defined as `flashCall` but, e.g.: https://github.com/sinch/sinch-sdk-python/blob/DEVEXP-205-186-verification-API/sinch/domains/verification/endpoints/get_verification_by_id.py#L26C13-L26C48
Will set the value to `flashcall` so if end user is using enum value for a test it will failed (and same for other classes)